server/server.go

```diff
@@ -27,14 +27,39 @@ import (
 	"github.com/ava-labs/avalanchego/message"
 	"github.com/ava-labs/avalanchego/snow/networking/router"
 	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"go.uber.org/zap"
+	"golang.org/x/exp/maps"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/status"
 )
 
+const (
+	MinNodes     uint32 = 1
+	DefaultNodes uint32 = 5
+
+	stopTimeout           = 5 * time.Second
+	defaultStartTimeout   = 5 * time.Minute
+	waitForHealthyTimeout = 2 * time.Minute
+
+	rootDataDirPrefix = "network-runner-root-data"
+)
+
+var (
+	ErrInvalidVMName          = errors.New("invalid VM name")
+	ErrInvalidPort            = errors.New("invalid port")
+	ErrNotEnoughNodesForStart = errors.New("not enough nodes specified for start")
+	ErrNetworkRunning         = errors.New("network already running")
```

Comment:
I don't think we need to change Err names on err description. This errors where already avaiable with different naming. And they are also part of interface with clients.

Reply:
I don't think we had errors for "network already running" or "no network is running"

Reply:
we had ErrAlreadyBootstrapped and ErrNotBootstrapped with same semantics. Would prefer to keep them until bigger change to API.